content/docs/reference-react.md

```diff
@@ -71,7 +71,7 @@ See the [React.Component API Reference](/docs/react-component.html) for a list o
 
 ### `React.PureComponent`
 
-`React.PureComponent` is exactly like [`React.Component`](#reactcomponent), but implements [`shouldComponentUpdate()`](/docs/react-component.html#shouldcomponentupdate) with a shallow prop and state comparison.
+`React.PureComponent` is exactly like [`React.Component`](#reactcomponent), but implements [`shouldComponentUpdate()`](/docs/react-component.html#shouldcomponentupdate) with a shallow prop and state comparison, which [`React.Component`](#reactcomponent) is missing completely. 
```

Comment:
Maybe it would be clearer if it was something like:

>[PC] is similar to [C]. The difference between them is that [C] doesn't implement [sCU], but [PC] ...

?

Reply:
Makes total sense for me. Updated my PR.